### PR TITLE
SC-5987 - SC-5410 language switcher in nuxt. You can find it on user account page

### PR DESCRIPTION
**Pull request by mwolowyk-cap**
# Description

- 5987 - language switcher for nuxt located in user account
- 5410 - added dropdown in account page for selecting user language

## Links to Tickets or other pull requests

- https://github.com/hpi-schul-cloud/schulcloud-server/pull/1679
- https://ticketsystem.schul-cloud.org/browse/SC-5987
- https://ticketsystem.schul-cloud.org/browse/SC-5410

## Screenshots of UI changes
image
image
image
image

### CODE REVIEW

**Review by DanielRieckhof — APPROVED** [review]

**Review by DanielRieckhof — APPROVED** [review]

File: src/layouts/legacyLoggedIn.vue [hunk 1]
```diff
@@ -12,6 +12,7 @@
 					:title="pageTitle"
 					:actions="topBarActions"
 					:fullscreen-mode="fullscreenMode"
+					:default-language="defaultLanguage"
```
Comment (mwolowyk-cap) [inline src/layouts/legacyLoggedIn.vue:15]: check if it is necessary here

File: src/pages/account/index.vue [hunk 2]
```diff
@@ -111,6 +111,25 @@
...
+			:heading="$t('pages.account.index.user.locale')"
+			:data="currentUser.defaultLanguage"
+			:target-path="`/account/locale/edit`"
+			data-testid="account_card_locale"
+		>
+			<template v-slot:icon>
+				<base-icon
+					source="material"
+					icon="language"
+					style="
```
Comment (mwolowyk-cap) [inline src/pages/account/index.vue:124]: we should use class instead

**Comment by ABialoskorski** [comment]
@schul-cloud-bot update screenshots

**Comment by mwolowyk-cap** [comment]
@schul-cloud-bot update screenshots

**Comment by ABialoskorski** [comment]
@schul-cloud-bot update screenshots

**Review by Metauriel — APPROVED** [review]
looks good to me, but please check michaels comments

File: src/layouts/legacyLoggedIn.vue (hunk 1 above)
Comment (ABialoskorski) [reply src/layouts/legacyLoggedIn.vue]: deleted

File: src/pages/account/index.vue (hunk 2 above)
Comment (ABialoskorski) [reply src/pages/account/index.vue:124]: done ;)

**Review by DanielRieckhof — APPROVED** [review]

**Pull request merged**